src/view/OpenRobertaWindow.vala

```diff
@@ -0,0 +1,99 @@
+/*
+ * brickman -- Brick Manager for LEGO MINDSTORMS EV3/ev3dev
+ *
+ * Copyright 2015 Stefan Sauer <ensonic@google.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * OpenRobertaWindow.vala - Edit connection status
+ *
+ * If not connected (initiallly)
+ * - A list of entries:
+ *   lab.open-roberta.org
+ *   10.0.1.10:1999 (usb?)
+ *   user-definable
+ *   Another (type in)
+ *
+ * If selected 'another'
+ * - dialog to enter ip+port
+ * - we could be clever and suggest the network part of the ev3's ip already and
+ *   have the port ":1999" there too
+ *
+ * When selected an address or after typing the new address
+ * - service.connect(address) and show pairing code
+ *
+ * Once connected
+ * - show disconnect button
+ *
+ * Once disconnected or if connection lost
+ * - go back to start
+ */
+
+using Ev3devKit;
+using Ev3devKit.Ui;
+
+namespace BrickManager {
+    public class OpenRobertaWindow : BrickManagerWindow {
+        internal Label info;
```

Comment:
I get not including the type name in variable name just for the sake of including the type, but in the case of UI widgets I always include it because it describes what the variable is. For example, the `*_server` variables below do not represent a server, they represent a menu item, so I like the variable name to be `*_menu_item`.